lib/hammer_cli_foreman/config_report.rb

```diff
@@ -1,8 +1,8 @@
 module HammerCLIForeman
 
-  class Report < HammerCLIForeman::Command
+  class ConfigReport < HammerCLIForeman::Command
```

Comment:
@ofedoren, there is still reference to `Report` in hammer_cli_foreman.rb L105. The "global" report command there should be also renamed. 

I agree with Tomas's point about keeping the report command as deprecated for a version or two.